### PR TITLE
#76 - Create a Past Events page

### PR DESCRIPTION
**Pull request by ericpoirier**
Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by ericpoirier** [comment]
> This PR is causing a regression for news on the homepage:
> https://deploy-preview-87--eclipsefdn-openadx.netlify.app/

For the news or Events section?

File: content/events/2020-02-28.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 ---
 title: "Conference Automotive Software Strategies"
-date: 2020-02-28T16:27:43+00:00
```
Comment (chrisguindon) [inline content/events/2020-02-28.md:3]: The prefered date format is 2020-02-28T16:27:43+00:00 not 2020-03-03

File: layouts/index.html [hunk 2]
```diff
@@ -23,21 +23,27 @@ <h2 class="header-underline margin-bottom-30">News</h2>
...
+              {{ $t := time .Params.end_date }}
+              {{ if $t.After now }}
+              <div class="row news-item">
+                <div class="col-sm-16">
+                  <p class="margin-bottom-0"><strong>{{ .Title }}</strong></p>
+                  <p class="margin-bottom-0">{{ .Params.location }}</p>
+                  <p>{{ .Params.event_date }}</p>
+                </div>
+                <div class="col-sm-8">
+                  <p class="events-list-link text-right"><a class="btn btn-secondary" href="{{- with .Params.link | default .RelPermalink }} {{.}} {{- end }}">Learn more</a></p>
```
[the remark this answers is not in the document]
Comment (chrisguindon) [reply layouts/index.html:37]: I agree it's not nessasery but it's probably a copy and paste from another one of our site.

[the remark this answers is not in the document]
Comment (ericpoirier) [reply layouts/index.html:37]: I believe this is there to get the default link or the custom link from the page.

[the remark this answers is not in the document]
Comment (chrisguindon) [reply layouts/index.html:37]: > > This PR is causing a regression for news on the homepage:
> > https://deploy-preview-87--eclipsefdn-openadx.netlify.app/
> 
> For the news or Events section?

All good! I did not notice at first that you made an update to the homepage to hide past event.

**Comment by chrisguindon** [comment]
> > This PR is causing a regression for news on the homepage:
> > https://deploy-preview-87--eclipsefdn-openadx.netlify.app/
> 
> For the news or Events section?

Create a Past Events page

**Comment by chrisguindon** [comment]
> > This PR is causing a regression for news on the homepage:
> > https://deploy-preview-87--eclipsefdn-openadx.netlify.app/
> 
> For the news or Events section?

All good now! I did not notice at first that you made a change to the homepage to hide past events!

File: content/events/2020-02-28.md (hunk 1 above)
Comment (ericpoirier) [reply content/events/2020-02-28.md:3]: I updated the dates to match this format.

**Comment by chrisguindon** [comment]
@ariexi What is your preference here? Should we close this or would you prefer we create an event archive on this site?

**Comment by ariexi** [comment]
> 
> 
> @ariexi What is your preference here? Should we close this or would you prefer we create an event archive on this site?

Good question! 
Will the archive contain all past events including the events from the new events site or only the event from the WG page like `content/events/2020-02-28.md`?

Did I understand it right, that an archive will come in the future for every WG?
If yes, then I will wait and we can close this issue.
Thx

**Comment by chrisguindon** [comment]
These are the events that we have in our system right now:
https://newsroom.eclipse.org/api/events?parameters[publish_to][]=openadx

We would need someone to submit the missing events via https://newsroom.eclipse.org/node/add/events.

I posted some information about this here: https://github.com/EclipseFdn/events.eclipse.org/issues/15#issuecomment-633633695

**Comment by ariexi** [comment]
> These are the events that we have in our system right now:
> [https://newsroom.eclipse.org/api/events?parameters[publish_to][]=openadx](https://newsroom.eclipse.org/api/events?parameters%5Bpublish_to%5D%5B%5D=openadx)
> 
Yes, these are the current events.

> We would need someone to submit the missing events via https://newsroom.eclipse.org/node/add/events.
> 
If this means, I should create the past events, I will do this, if this will make no problems.

> I posted some information about this here: [EclipseFdn/events.eclipse.org#15 (comment)](https://github.com/EclipseFdn/events.eclipse.org/issues/15#issuecomment-633633695)

**Comment by chrisguindon** [comment]
@shanda-eclipse are you good with @ariexi submitting past events to Eclipse Newsroom?

**Comment by shanda-eclipse** [comment]
> @shanda-eclipse are you good with @ariexi submitting past events to Eclipse Newsroom?

Yep. They can be submitted through the newsroom and I will pass them through moderation.

**Comment by ariexi** [comment]
I found a strange behaviour on my mobile, when opening OpenADx website on Safari.

I will send you the picture by mail, because I cannot upload files here...

**Comment by chrisguindon** [comment]
@ariexi are the news better now?

**Comment by ariexi** [comment]
> 
> 
> @ariexi are the news better now?

@chrisguindon , yes, it is looking great now!
Thank you so much!
Best,
Andy

**Comment by chrisguindon** [comment]
Closing this PR in favor of PR #95

**Pull request closed without merging**